CMakeLists.txt

```diff
@@ -264,6 +264,28 @@ foreach(RELTYP RELEASE DEBUG RELWITHDEBINFO MINSIZEREL)
         endforeach()
 endforeach()
 
+#
+# Handling of LTO/IPO
+#
+
+option(DISABLE_LTO "Disable use of LTO when building. Defaults to being enabled if supported." FALSE)
```

Comment:
As a general tip, I prefer using `ENABLE_XXX` instead of `DISABLE_XXX`, because this avoids double negation in the code, eg. `NOT DISABLE_XXX`.